src/backend/egl/context.rs

```diff
@@ -37,6 +40,39 @@ pub struct EGLContext {
 unsafe impl Send for EGLContext {}
 
 impl EGLContext {
+    /// Creates a new `EGLContext` from raw handles.
+    ///
+    /// # Safety
+    ///
+    /// - Not using the system default EGL library (`dlopen("libEGL.so")`) as loaded by smithay might cause undefined behavior
```

Comment:
Stylistic changes to use less passive language, the general idea is fine

```suggestion
    /// - The context must be created from the system default EGL library (`dlopen("libEGL.so")`)
```